src/components/cc-email-list/cc-email-list.js

```diff
@@ -0,0 +1,404 @@
+                      danger
+                      outlined
+                      image=${trashSvg}
+                      @cc-button:click=${() => this._onDelete(secondaryAddress.address)}
+                      ?waiting="${secondaryAddress.state === 'deleting'}"
+                      ?disabled="${busy}"
+                      accessible-name="${i18n('cc-email-list.secondary.action.delete.accessible-name', { address: secondaryAddress.address })}"
+                  >
+                    ${i18n('cc-email-list.secondary.action.delete.name')}
+                  </cc-button>
+                </div>
+              </li>
+            `;
+          })}
+        </ul>
+
+        ${this._renderAddEmailForm()}
+      </cc-block-section>
+    `;
+  }
+
+  _renderAddEmailForm () {
+    const isAdding = this.addEmailForm.state === 'adding';
+
+    return html`
+      <form>
+        <cc-input-text
+            label="${i18n('cc-email-list.secondary.address-input.label')}"
+            required
+            .value="${this.addEmailForm.address.value}"
+            ?disabled=${isAdding}
+            @cc-input-text:requestimplicitsubmit=${this._onAdd}
+            ${ref(this._addressInputRef)}
+        >
+          ${this._renderAddressError()}
+          <p slot="help">
+            ${i18n('cc-email-list.secondary.address-input.format')}
+          </p>
+        </cc-input-text>
+        <cc-button
+            primary
+            ?waiting=${isAdding}
+            @cc-button:click=${this._onAdd}
+        >
+          ${i18n('cc-email-list.secondary.action.add')}
+        </cc-button>
+      </form>
+    `;
+  }
+
+  _renderAddressError () {
+    if (this.addEmailForm.address.error === 'empty') {
+      return html`<p slot="error">${i18n('cc-email-list.secondary.address-input.error.empty')}</p>`;
+    }
+    if (this.addEmailForm.address.error === 'used') {
+      return html`<p slot="error">${i18n('cc-email-list.secondary.address-input.error.used')}</p>`;
+    }
+    if (this.addEmailForm.address.error === 'invalid') {
+      return html`<p slot="error">${i18n('cc-email-list.secondary.address-input.error.invalid')}</p>`;
+    }
+    if (this.addEmailForm.address.error === 'already-defined') {
+      return html`<p slot="error">${i18n('cc-email-list.secondary.address-input.error.already-defined')}</p>`;
+    }
+
+    return html``;
+  }
+
+  static get styles () {
+    return [
+      skeletonStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        ul.secondary-addresses {
+          margin: 0;
+          padding: 0;
+        }
+
+        /*region address-line*/
+
+        .address-line {
+          align-items: center;
+          display: flex;
+          flex-wrap: wrap;
+          gap: 1em;
+        }
+
+        .address-line .address.loading {
+          opacity: var(--cc-opacity-when-disabled);
+        }
+
+        .address-line .address span {
+          min-width: 15em;
```

Comment:
In fact, I just want this for secondary addresses to avoid this:
![image](https://user-images.githubusercontent.com/104012218/202666680-d9d57084-5a58-4914-aca1-60b3fec3d736.png)
By the way, if you have a better solution to handle that, I'd be glad to hear about it.